libs/repositories/src/repos/UsdRepository/UsdRepositoryCow.ts

```diff
@@ -71,6 +94,22 @@ export class UsdRepositoryCow extends UsdRepositoryNoop {
     return usdcPrice.div(tokenPrice).toNumber();
   }
 
+  private getChainId(chainIdOrSlug: string) {
+    // Only SupportedChainIds are supported
+    const numericId = isNaN(+chainIdOrSlug)
+      ? SUPPORTED_CHAIN_SLUG_TO_ID[chainIdOrSlug]
+      : (+chainIdOrSlug as SupportedChainId);
+
+    if (!SupportedChainId[numericId]) {
+      logger.debug({
+        msg: `Chain ${chainIdOrSlug} not supported on UsdRepositoryCow`,
```

Comment:
Why logging this? we know we don't support this. Is expected. This will just created noise

Reply:
Agree, removed that